FestivalPlanner/templates/theaters/theaters.html

```diff
@@ -0,0 +1,45 @@
+{% extends "base_template.html" %}
+
+{% block title %}{{ title }}{% endblock %}
+{% block header %}{{ title }}{% endblock %}
+
+{% block content %}
+    <h3>Theaters Overview</h3>
+    {% if log %}
+        <h3 style="color:{{ festival_color }}; margin-bottom: 2px">{{ log.action }} results</h3>
+        {% for result in log.results %}
+            <p class="log">{{ result }}</p>
+        {% endfor %}
+    {% endif %}
+    <br>
+    {% if theater_rows %}
+        <table>
+            <thead style="position:sticky; top:14pt; background-color: #00000090;">
```

Comment:
Please move this style to each column header, so that the table grid remains visible in the header.

You might want to use a class to make this more simple, as suggested in an earlier comment.

Reply:
Done.